.github/workflows/format.yml

```diff
@@ -0,0 +1,17 @@
+name: Format
+
+on:
+  pull_request:
+  push:
+
+jobs:
+  format:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.18
```

Comment:
Keto already is on go1.19

Reply:
Good to know, thanks! I'm updating all codebases to Go 1.19 and see what breaks.